### DISCUSSION TITLE
Databricks source connector

### DISCUSSION THREAD

**Discussion by nohelia-merino** [airbytehq/airbyte#36125] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination? Databricks source connector
* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB?  No specific version

## Describe the context around this new connector
Need to transfer data from Databricks to AWS S3 bucket

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
Databricks mounting feature

**Comment by mtrienis** [comment]
Hi, I'm looking to setup databricks to braze connection. Any progress on this issue?
